dotnet/hello_world/Program.cs

```diff
@@ -0,0 +1,210 @@
+﻿using Google.Cloud.Bigtable.Admin.V2;
+using Google.Cloud.Bigtable.V2;
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Google.Protobuf;
+using System.Linq;
+using System.IO;
+using System;
+
+
+namespace HelloWorld
```

Comment:
Braces on the same line as previous line, please.

Reply:
https://softwareengineering.stackexchange.com/questions/236995/why-do-c-developers-newline-opening-brackets

The default setting in Visual studio is on a new line